charter/index.html

```diff
@@ -433,30 +435,30 @@ <h3 id="w3c-coordination">W3C Groups</h3>
             <dd>To maintain a separation of goals and transfer of incubated work items.</dd>
 
             <dt><a href="https://www.w3.org/groups/cg/unhosted">Unhosted Web Community Group</a></dt>
-            <dd>To explore the overlaps (CORS, user-centric storage, JSON-LD container descriptions) and differences (OAuth instead of WAC/ACP) between the Solid Protocol draft and their work on <a href="https://datatracker.ietf.org/doc/draft-dejong-remotestorage/">remoteStorage</a>.</dd>
+            <dd>To explore the overlaps (CORS, user-centric storage, JSON-LD container descriptions) and differences (OAuth instead of WAC/ACP) between the PUMPKIN Protocol draft and their work on <a href="https://datatracker.ietf.org/doc/draft-dejong-remotestorage/">remoteStorage</a>.</dd>
```

Comment:
```suggestion
            <dd>To explore the overlaps (e.g., CORS, user-centric storage, JSON-LD container descriptions) and differences (e.g., OAuth instead of WAC/ACP) between the PUMPKIN Protocol draft and their work on <a href="https://datatracker.ietf.org/doc/draft-dejong-remotestorage/">remoteStorage</a>.</dd>
```